### PR TITLE
cw3-fixed-multisig: write cw20 multi-contract mint test

### PR DESCRIPTION
**Pull request by orkunkl**
This PR is for multi-contract testing framework workshop.
Also will increase coverage.

### CODE REVIEW

**Comment by maurolacy** [comment]
Hi,

Can you add some context / description?
Also, this mentions `flex` in the title and branch, but the changes are in the `cw3-fixed-multisig` contract.

File: contracts/cw3-fixed-multisig/src/integration_tests.rs [hunk 1]
```diff
@@ -0,0 +1,144 @@
+#![cfg(test)]
```
Comment (maurolacy) [inline contracts/cw3-fixed-multisig/src/integration_tests.rs:1]: If this is meant as a "classic" integration test (i. e. to be compiled to wasm, and run in the VM), it must go in `tests/` directory, and called `integration.rs`. Then you can do `cargo wasm && cargo integration-test` to run it.

Besides, you'll have to modify it to run in the VM context. That's relatively straightforward to do. Take a look at the header comments in any of the integration tests in the `cosmwasm` repo.

**Comment by orkunkl** [comment]
@maurolacy You are right about the naming, I got distracted when creating the PR. I will fix the commit when squashing the commits.

File: contracts/cw3-fixed-multisig/src/integration_tests.rs (hunk 1 above)
Comment (ethanfrey) [reply contracts/cw3-fixed-multisig/src/integration_tests.rs:1]: Another name would be better, but it does belong here. 
It runs in a (mock) multi-contract environment. Actually more useful feedback than vm integration tests in may cases as you capture contract-contract interactions.

No need to move it, maybe just rename it

**Comment by ethanfrey** [comment]
@orkunkl Can you polish this off so we can merge. I want to do a big cosmwasm-plus upgrade to v0.14.0 soon, and this should be in master first. (Well, I don't really want to do the upgrade but need it done so I can write some contracts for hackatom ru... any volunteers to bump cosmwasm?)

**Comment by ethanfrey** [comment]
Hmm... I'll just merge this in cuz Mauro started the upgrade. But please feel free to polish it off later.

**Pull request merged**